src/docs/pages/introduction/getting-started/vue.md

```diff
@@ -0,0 +1,109 @@
+---
+
+# Vue
+
+Liquid Oxygen comes with [Vue](https://vuejs.org/) bindings for all Web Components. Although the rendered components are still Web Components, the bindings improve the developer experience by providing a more familiar API and a better integration with Vue.
+
+On this page, you'll find detailed instructions on how to integrate Liquid Oxygen into your Vue project and how to use the components.
+
+## Prerequisites
+
+Liquid Oxygen is easy to integrate into an existing Vue project. We assume you already have a project set up. If not, you can create a new project with [Vite](https://vitejs.dev/).
+
+```sh
+npm create vite@latest your-project-name -- --template vue-ts
+```
+
+For more information about this command and Vite, please refer to the [Vite documentation](https://vitejs.dev/guide/#scaffolding-your-first-vite-project).<br />Although this guide assumes your project is using typescript, all examples should also be applicable to javascript projects.
+
+## Install
+
+Add Liquid Oxygen to your project with the package manager of your choice.
+
+```sh
+npm install @emdgroup-liquid/liquid
+```
+
+<ld-notice>
+  Liquid Oxygen works with any package manager (e.g. <code>npm</code>, <code>yarn</code> or <code>pnpm</code>). For simplicity, we use <code>npm</code> in this guide.
+</ld-notice>
+
+### Import stylesheet
+
+All Web Components are loaded togehter with their styles embedded. Therefore we only need to import the global stylesheet for Liquid Oxygen.
+
+Add the following code to your `App.vue` file (or any similar file which is loaded for every page).
+
+```html
+// App.vue
+<script setup lang="ts">
+  import '@emdgroup-liquid/liquid/dist/css/liquid.global.css'
+</script>
+```
+
+## Usage
+
+When adding Liquid Oxygen components to a Vue project, it is crucial to use the Vue bindings. All components are imported from `@emdgroup-liquid/liquid/dist/vue`. The bindings significantly improve Vue compatibility and your developer experience.
+
+Let's have a look at how to add a [LdButton](components/ld-button/) to your project. This examnple also includes a [LdIcon](components/ld-icon/) as it helps you to check if Liquid Oxygen assets are loaded correctly.
+
+```html
+// SampleComponent.vue
+<script lang="ts">
+import { LdButton, LdIcon } from '@emdgroup-liquid/liquid/dist/vue'
+<script>
+
+<template>
+  <LdButton>
+    Click me!
+    <LdIcon name="energy" />
+  </LdButton>
+</template>
+```
+
+When you put this component on a page, you should see a blue button with the text "Click me!" and a lightning bolt icon.
+
+### Events
+
+Liquid Oxygen components aim to work similarly to native HTML elements as much as possible. In most cases, you can expect the same events and behavior from a Liquid Oxygen component and its native equivalent. Custom events are documented on the respective component pages.
+
+Let's take our button from above and add a click handler.
+
+```html
+// SampleComponent.vue
+<script lang="ts">
```

Comment:
This is not quite how a typical component looks like in Vue. You would rather use a [method handler](https://vuejs.org/guide/essentials/event-handling.html#method-handlers). [Here is a nice complete example](https://sfc.vuejs.org/#eNp9UV1LwzAU/SvXILjB1ryXOhQEfVHffMpDs/auzWyTktxWYfS/m2RLEQdCaXK/zrk558QehyGbRmQ5K1xl1UA7ofF7MJagxoMcO4KT0AC1JLlan+8AFmm0OkUAWvaYw93HiNnR3Z2zczjmTfj3SK2pXZ4GGotIK5xQ0wIJwDmU1CpXgtJO1ZjGYDBKkwMyQC1CNVrrB0FWpCYMvSR1hQlFdmhpVb5g1xm4PQXALKw335Tr30yR3VN5XA+qZQR7en+FWEid6gBXeyaOmM9I2gbD0bx5loUivj6JILT/Cr4I7APCfugkYYio2I9ERsND1anq816wKJBgu+dwFvxc9q0FX+bYhqk++LTt5eBVN9p7GHcUl4ITbFFcMG9yiAVriQaXc+4OVXD+6DJjG+5vmR01qR4zdP12b82XQ+uBBYseXjC4T05ot96CGi3a/zD/tF7hJm3Y/AOPytj0).